packages/typespec-ts/test/modularIntegration/customClientRequestId.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { PipelinePolicy } from "@azure/core-rest-pipeline";
+import { RequestIdClient } from "./generated/headers/client-request-id/src/index.js";
+import { assert } from "chai";
+describe("RequestIdClient Classical Client", () => {
+  let client: RequestIdClient;
+
+  beforeEach(() => {
+    client = new RequestIdClient({
+      allowInsecureConnection: true
+    });
+  });
+
+  it("should add client-request-id in header transparently", async () => {
+    try {
+      const result = await client.get();
+      assert.isUndefined(result);
```

Comment:
why this should be undefined ? is it a normal call to cadl-ranch ?

Reply:
This is a normal call to cadl-ranch and its validation is for client request id in header. And the return type in modular would be void. If this call could be return directly without any exception and the validation passed.